force-app/main/default/aura/FileListView/FileListViewController.js

```diff
@@ -0,0 +1,69 @@
+({
+	doInit : function(component, event, helper) {      
+        helper.getColumns(component);
+        helper.getDocuments(component, helper);
+        helper.getSelectOptions(component);
+        helper.getTotalCount(component);
+	},    
+    handleRowAction : function(component, event, helper) {
+    	var selRows = event.getParam('selectedRows');
+        component.set("v.delIds",selRows);
+        var btnDelete = component.find("btnDelete");
+        if(selRows.length === 0){                            
+        	btnDelete.set('v.disabled',true);
+       	}
+        else{
+        	btnDelete.set('v.disabled',false);
+        }        
+    },
+    doDelete : function(component, event, helper){
+        if(confirm('Are you sure you want to delete these items?')) {
+        	var delIdList = component.get("v.delIds");
+        	var action = component.get('c.deleteItems');
+        	action.setParams({lstId : delIdList});
+        	action.setCallback(this, function(response) {
+            var state = response.getState();
+            if (state === "SUCCESS") {
+                $A.get('e.force:refreshView').fire();
+                alert('Successfully Deleted');   
+            }
+            else if (state === "ERROR") {
+                var errors = response.getError();
+                if (errors) {
+                    if (errors[0] && errors[0].message) {
+                        console.log("Error message: " + errors[0].message);                                    
+                    }
+                } 
+                else {
+                    console.log("Unknown Error");
+                }
```

Comment:
Do you display the errors somewhere, if not displaying an alert with the error to the user would be useful.